starknet-foundry/crates/forge/tests/integration/roll.rs

```diff
@@ -169,3 +169,63 @@ fn start_roll_in_constructor_test() {
 
     assert_passed!(result);
 }
+
+#[test]
+fn stop_roll() {
+    let test = test_case!(
+        indoc!(
+            r#"
+            use result::ResultTrait;
+            use array::ArrayTrait;
+            use option::OptionTrait;
+            use traits::TryInto;
+            use starknet::ContractAddress;
+            use starknet::Felt252TryIntoContractAddress;
+            use cheatcodes::PreparedContract;
+            use forge_print::PrintTrait;
+            
+            #[starknet::interface]
+            trait IRollChecker<TContractState> {
+                fn get_block_number(ref self: TContractState) -> u64;
+            }
+
+            #[test]
+            fn test_stop_roll() {
+                let class_hash = declare('RollChecker');
+                let prepared = PreparedContract { class_hash: class_hash, constructor_calldata: @ArrayTrait::new() };
+                let contract_address = deploy(prepared).unwrap();
+                let contract_address: ContractAddress = contract_address.try_into().unwrap();
+                let dispatcher = IRollCheckerDispatcher { contract_address };
+            
+                start_roll(contract_address, 234);
+            
+                let block_number = dispatcher.get_block_number();
+                assert(block_number == 234, 'Wrong block number');
+
+                stop_roll(contract_address);
+
+                let block_number = dispatcher.get_block_number();
+                assert(block_number != 234, 'Block number  did not change back'');
```

Comment:
How does this work? The syntax seems wrong 😱 And the tests are passing? 🤔

Reply:
Maybe they changed the way they are parsing? Hmmm

Reply:
Should def be a syntax error

Reply:
Fixed, not sure how does this work though, we should make an issue for the investigation of this topic.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/338